utilities/cloudharness_utilities/utils.py

```diff
@@ -16,10 +16,13 @@
 )
 
 
-def app_name_from_path(dockerfile_path):
-    return get_image_name("-".join(p for p in dockerfile_path.split("/") if p not in NEUTRAL_PATHS))
+def image_name_from_dockerfile_path(dockerfile_path, base_name=None):
+    return get_image_name(app_name_from_dockerfile_path(dockerfile_path), base_name)
 
 
+def app_name_from_dockerfile_path(dockerfile_path):
```

Comment:
wasn't it easier to return the mainapp (from the values(-template).yaml) as the app_name?